apps/api/src/mail/mail.service.ts

```diff
@@ -1,16 +1,17 @@
-import { Inject, Injectable, Logger, LoggerService } from '@nestjs/common';
+import { Injectable } from '@nestjs/common';
 import * as path from 'path';
 import axios from 'axios';
 import * as fs from 'fs';
 import { stringify } from 'qs';
 import * as handlebars from 'handlebars';
+
 import { Mailable } from './mailables/mail-base.mailable';
 import { MailOptions } from './mail-options.interface';
 import { ChesResponse } from './types/ches-response';
 
 @Injectable()
 export class MailService {
-  constructor(@Inject(Logger) private readonly logger: LoggerService) {
+  constructor() {
```

Comment:
Do our application error filter provides enough context to debug different failure stages of this service?

Reply:
The original error will be caught in the controller and logged, so we'll know what the failed response was as well as what the original error is